wire-ios/Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+NavigationBar.swift

```diff
@@ -126,11 +158,19 @@ extension ConversationViewController {
         return button
     }
 
-    private func backButtonIcon(hasUnreadInOtherConversations: Bool) -> StyleKitIcon {
+    private func backButtonIcon(hasUnreadInOtherConversations: Bool) -> UIImage {
         if view.isRightToLeft {
```

Comment:
Have you tested both variants? I haven't started the app in a right-to-left mode yet.

Reply:
Yep! But in this case, we didn't do anything custom other than setting the icon.